app/tests/unit/components/Form/ProjectEmissionIntensityReportFormSummary.test.tsx

```diff
@@ -55,11 +55,19 @@ const defaultMockResolver = {
               isPristine: false,
               newFormData: {
                 reportingRequirementId: 1,
+                baselineEmissionIntensity: 0.87654321,
+                teimpReporting: {
+                  baselineEmissionIntensity: 0.87654321,
+                },
               },
               operation: "UPDATE",
               formChangeByPreviousFormChangeId: {
                 newFormData: {
                   reportingRequirementId: 1,
+                  baselineEmissionIntensity: 0.985145,
+                  teimpReporting: {
+                    baselineEmissionIntensity: 0.985145,
+                  },
```

Comment:
I know we can't properly test the three diffs until the proper styling is in, but it might be good here to test for the existence of a latest committed value to make sure it's coming through properly, since this is the PR where that functionality is added